core/entity/data/v1/v1_test.go

```diff
@@ -475,6 +512,34 @@ func TestSecretStored_Parse(t *testing.T) {
 			wantErr: false,
 			err:     nil,
 		},
+		{
+			name: "valid_values_json",
+			fields: fields{
+				Name:   "test-2",
+				Values: []string{"{\"pass\":\"secret\"}"},
+				Meta: SecretMeta{
+					Template: "{\"PASS\":\"{{.pass}}\"}",
+					Format:   "json",
+				},
+			},
+			want:    "{\"PASS\":\"secret\"}",
+			wantErr: false,
+			err:     nil,
+		},
+		{
+			name: "valid_values_json_passed_as_yaml",
```

Comment:
Are these tests redundant here? We have these test cases as part of transform() tests.  And parse() specifically doesn't do any validation around template and format. Please correct me if I'm missing something.

Reply:
Yes, I completely agree with your comment, this test case should be part of test cases for transform().